tests/test_browser.py

```diff
@@ -1576,6 +1576,13 @@ def test_egl_width_height(self):
   def test_egl_width_height_with_proxy_to_pthread(self):
     self._test_egl_width_height_base('-s', 'USE_PTHREADS', '-s', 'PROXY_TO_PTHREAD=1')
 
+  @requires_graphics_hardware
+  def test_egl_createcontext_error(self):
+    create_test_file('test_egl_createcontext_error.c', self.with_report_result(open(path_from_root('tests', 'test_egl_createcontext_error.c')).read()))
```

Comment:
I think these 3 lines can be replaced with `self.btest('test_egl_createcontext_error.c', '1', args=['-lEGL', '-lGL'])`